package.json

```diff
@@ -8,16 +8,19 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "bootstrap": "^5.0.1",
```

Comment:
one more thing, @carrotfarmer - [consider `bootstrap-vue`](https://bootstrap-vue.org/) instead of the vanilla one

Reply:
@floydpink bootstrap-vue does not support vue 3

Reply:
@carrotfarmer - I see an email with your comment saying Vue 3 is unsupported by Bootstrap-Vue - but cannot see that comment here on this PR! Weird.... 🙂 

But - since that is the case, we could also switch into something new and radically different like Tailwind CSS or some such.

Reply:
@floydpink Sorry that was my mistake the project runs Vue 2